scripts/zones/Temenos/mobs/Abyssdweller_Jhabdebb.lua

```diff
@@ -2,31 +2,34 @@
 -- Area: Temenos
 --  Mob: Abyssdweller Jhabdebb
 -----------------------------------
-require("scripts/globals/limbus");
------------------------------------
+mixins = {require("scripts/mixins/job_special")}
+local ID = require("scripts/zones/Temenos/IDs")
 
-function onMobEngaged(mob,target)
-  if   (IsMobDead(16929010)==true and IsMobDead(16929011)==true and IsMobDead(16929012)==true and
-        IsMobDead(16929013)==true and IsMobDead(16929014)==true and IsMobDead(16929015)==true
-    ) then
-       mob:setMod(tpz.mod.SLASHRES,1400);
-       mob:setMod(tpz.mod.PIERCERES,1400);
-       mob:setMod(tpz.mod.IMPACTRES,1400);
-       mob:setMod(tpz.mod.HTHRES,1400);
-  else
-      mob:setMod(tpz.mod.SLASHRES,300);
-      mob:setMod(tpz.mod.PIERCERES,300);
-      mob:setMod(tpz.mod.IMPACTRES,300);
-      mob:setMod(tpz.mod.HTHRES,300);
-  end
-  GetMobByID(16929006):updateEnmity(target);
-  GetMobByID(16929007):updateEnmity(target);
-end;
+function onMobEngaged(mob, target)
+    if GetMobByID(ID.mob.TEMENOS_C_MOB[3]+5):isDead() and GetMobByID(ID.mob.TEMENOS_C_MOB[3]+6):isDead() and
+        GetMobByID(ID.mob.TEMENOS_C_MOB[3]+7):isDead() and GetMobByID(ID.mob.TEMENOS_C_MOB[3]+8):isDead() and
+        GetMobByID(ID.mob.TEMENOS_C_MOB[3]+9):isDead() and GetMobByID(ID.mob.TEMENOS_C_MOB[3]+10):isDead()
+    then
+        mob:setMod(tpz.mod.SLASHRES, 1400)
+        mob:setMod(tpz.mod.PIERCERES, 1400)
```

Comment:
Same as before regarding strength/weaken and engaged v death, but this time to the players' detriment. 😄 

(In this case, being aggroed by a boss before the last mook is dead, and then the boss not getting weaker until it re-engages.)